store/test_schema/clickhouse/1_todo.sql

```diff
@@ -0,0 +1,24 @@
+-- Covered objects:
+-- - Table
+-- - Index
+
+CREATE DATABASE bytebase_test_todo;
+
+-- Table and Index
+CREATE TABLE bytebase_test_todo.author (
+    id UUID,
+    name String
+) ENGINE = MergeTree()
+ORDER BY id;
+
+CREATE TABLE bytebase_test_todo.todo (
+    id UUID,
+    name String COMMENT 'name of the todo',
+    content String,
+    author_id UUID,
+    created_ts BIGINT NOT NULL,
+    updated_ts Int64 NOT NULL,
```

Comment:
why Int64 for updated_ts, BIGINT for created_ts?

Reply:
BIGINT is a alias for Int64 in clickhouse. I'll replace BIGINT with Int64 to stick with data type definitions in clickhouse documentation.